openrouteservice/src/main/resources/app.config.sample

```diff
@@ -68,6 +68,7 @@
             "maximum_distance_alternative_routes": 100000,
             "maximum_alternative_routes": 3,
             "maximum_distance_round_trip_routes": 100000,
+            "maximum_speed": 80,
```

Comment:
The name `maximum_speed` might not entirely reflect the purpose of the parameter. In fact, it might falsely indicate a globally enabled speed limit. I would rename it to sth more descriptive such as `maximum_speed_lower_bound`.